ppb/systems/renderer.py

```diff
@@ -134,9 +136,12 @@ def __init__(
         self.window = None
         self.window_title = window_title
         self.pixel_ratio = None
-        self.render_clock = 0
+        self.render_clock = time()
```

Comment:
Can you use the monotonic clock instead of the irl civil wall time? Unless there's a reason to not use monotonic.

Reply:
✔️